src/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -869,6 +877,10 @@ private SecurityStatusPal GenerateToken(byte[] input, int offset, int count, ref
             byte[] alpnResult = SslStreamPal.GetNegotiatedApplicationProtocol(_securityContext);
             _negotiatedApplicationProtocol = alpnResult == null ? default : new SslApplicationProtocol(alpnResult, false);
 
+            if (NetEventSource.IsEnabled)
+            {
+                NetEventSource.Exit(this);
+            }
```

Comment:
The corresponding Enter is under an ifdef for  TRACE_VERBOSE.  Either this should be as well, or that ifdef should be removed.

Reply:
Thanks for pointing this out! Will add TRACE_VERBOSE for this event.

Reply:
We should remove the TRACE_VERBOSE stuff.  It is legacy when we ported from .NET Framework.  It isn't very useful in CoreFx now.

Reply:
Opened #31227. I can submit a subsequent PR to remove all occurrences in CoreFx repo.

Reply:
I'm not saying we should just remove the ifdef TRACE_VERBOSE without understanding about the logging that is under it.

In terms of this PR, though, this particular set of traces should not be under TRACE_VERBOSE and that is what I was trying to comment on.

Reply:
Sorry that I misunderstood the comment. I'm not sure what type of logging should be under TRACE_VERBOSE , so I continue to use the style from the corresponding Enter.

FWIW, the `TRACE_VERBOSE` is introduced to replace `TRAVE` (which looks like a typo for `TRACE`), when porting the code from Framework. (#3163)

I can remove it in this case, since the logging happens during handshake procedure (don't have much overhead), and TRACE_VERBOSE  is not very useful in CoreFx.